nunit/MVP-Advance/SeleniumNUnit/Tests/Tests.cs

```diff
@@ -85,7 +85,40 @@ public void TC3c_WhenIAddInvalidDataThenIAssert()
             AssertInvalidData(10, 11, 12, "NegativeTC"); //need test data
         }
 
- 
+        [Test]
+        public void InvalidLang_1()
```

Comment:
for future ref keep in mind you can name tests in a more descriptive, i should be able to read the title of the and know what its testing

Reply:
I have done name changing in more descriptive